Assets/Scripts/Common/World/WorldGenerator.cs

```diff
@@ -0,0 +1,327 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace ubv.common.world
+{
+    public struct BoxInfo
+    {
+        Vector2Int m_coord;
+        Vector3Int m_info;
+
+        public BoxInfo(int x, int y, int width, int height) : this()
```

Comment:
Pourquoi le call à `this()` ?

Reply:
je sais, il l'a mis tout seul je vais l'enlever